src/total-lines-changed/total-lines-changed.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+// @jsx h
+
+import { h } from 'dom-chef'
+import elementReady from 'element-ready'
+import api from '../api'
+
+import './total-lines-changed.css'
+
+const FILES_TAB_SELECTOR = '[data-testid="sidebar-tab-files"]'
+
+let totalAdded: number
+let totalRemoved: number
```

Comment:
Is it necessary for this to be out here at module scope level? Can't it be within the scope of the function ?

If you navigate from one PR to a different one, is the count accurate in both ? I don't see these values being reset anywhere